packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -621,6 +621,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 						clientIdOrDocumentId === this.documentId ||
 						clientIdOrDocumentId === this.clientId;
 					const { code, type, message, retryAfter } = nacks[0]?.content ?? {};
+					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0].operation ?? {};
```

Comment:
Should this be `nacks[0]?.operation ??` ?

Reply:
```suggestion
					const {clientSequenceNumber, referenceSequenceNumber} = nacks[0]?.operation ?? {};
```

Reply:
yeah. i hate how array typing assumes indexes are non-null